src/configure/helper/commonHelper.ts

```diff
@@ -7,10 +7,7 @@ export async function sleepForMilliSeconds(timeInMs: number): Promise<void> {
 }
 
 export function generateDevOpsOrganizationName(userName: string, repositoryName: string): string {
-    let repoParts = repositoryName.split("/");
```

Comment:
Was this split to remove githib org name?

Reply:
yes it was to remove org name. as per our offline discussion yesterday, we are going ahead with userName-orgName-repoName